docker-compose/run-test.sh

```diff
@@ -2,20 +2,26 @@
 
 kuzzle=${KUZZLE_HOST:-kuzzle:7511}
 
-while ! curl -m 2 -silent -output /dev/null http://$kuzzle/api/1.0 > /dev/null
```

Comment:
... but I think we need it to ensure the integration tests to start only once Kuzzle is up.